Makefile

```diff
@@ -67,7 +67,7 @@ $(BOWER_MODULES): bower.json $(NODE_MODULES)
 lint: $(NODE_MODULES)
 	$(NODE) ./node_modules/.bin/jshint app.js $(shell find lib static -name '*.js' -not -path 'static/ext/*' -not -path static/analytics.js)
 
-LANG:=C++
+LANG:=haskell
```

Comment:
Please rever this change, defuak language should be C++. To launch your local instance with haskell, call `make EXTRA_ARGS="--language Haskell"` instead of changing this line

Reply:
ahh, thank you! I didn't know how to invoke this with `haskell` :)